### DISCUSSION TITLE
Couldn't generate link due to missingconnection URl

### DISCUSSION THREAD

**Discussion by XiaoHanL** [WalletConnect/walletconnect-monorepo#4578] [Developer Support]
！[上传微信IMG22.jpg...]()
I Couldn't generate link due to missingconnection URl when I was using it. Why is that? How to solve

**Comment by aoaohong** [comment]
Similarly, only the wallet can be connected, while other wallets prompt "I Couldn't generate link due to missingconnection URl"
Seeking a solution
